app/src/ui/lib/input-description/input-description.tsx

```diff
@@ -100,7 +100,7 @@ export class InputDescription extends React.Component<IInputDescriptionProps> {
    * */
   private getRole() {
     if (
-      InputDescriptionType.Error &&
+      this.props.inputDescriptionType === InputDescriptionType.Error &&
```

Comment:
This also happens a few lines above in `renderAriaLiveContainer` with `InputDescriptionType.Caption` 😅

Reply:
🤦 .. it was all of the place... must have been half asleep that day. :D